website/src/app/kb/deploy/clients/readme.mdx

```diff
@@ -28,13 +31,157 @@ below. If you find an exception, please
+| `authURL`                 | `String`  | `https://app.firezone.dev`         | The base auth URL of the Firezone application to sign in to. The `accountSlug` will be appended to this to form the complete sign-in URL.                                                                                                                                                          | macOS, iOS, Windows | 1.5.0           |
+| `apiURL`                  | `String`  | `wss://api.firezone.dev`           | The WebSocket URL of the Firezone control plane.                                                                                                                                                                                                                                                   | macOS, iOS, Windows | 1.5.0           |
+| `logFilter`               | `String`  | `info`                             | The [`RUST_LOG`-formatted](https://rust-lang-nursery.github.io/rust-cookbook/development_tools/debugging/config_log.html) log filter to apply to the connectivity library logger. Increasing the log level here can help troubleshoot connectivity issues at the cost of increased log file sizes. | macOS, iOS, Windows | 1.5.0           |
+| `accountSlug`             | `String`  | `<empty>`                          | The account slug or ID of your Firezone account.                                                                                                                                                                                                                                                   | macOS, iOS, Windows | 1.5.0           |
+| `startOnLogin`            | `Boolean` | `false`                            | Whether the client should start automatically on login.                                                                                                                                                                                                                                            | macOS, iOS, Windows | 1.5.0           |
+| `connectOnStart`          | `Boolean` | `false`                            | Whether the client should connect automatically on start.                                                                                                                                                                                                                                          | macOS, iOS, Windows | 1.5.0           |
+| `disableUpdateCheck`      | `Boolean` | `false`                            | Whether to disable the periodic update checker. The update checker is enabled by default for Windows, Linux, and standalone macOS Clients.                                                                                                                                                         | macOS, iOS, Windows | 1.5.0           |
```

Comment:
Called `checkForUpdates` on Windows.

Reply:
I was hoping to avoid needing separate MDM pages for each client, but it looks like we've already strayed far enough for that to be the case. I'll see what I can do.

Reply:
Yeah it is annoying but I don't really have a good solution for it. If the GUIs for the plist builders support it, maybe we can rename this one to `checkForUpdates` everywhere and make it a drop-down with `true` and `false`?

Reply:
Intune unfortunately doesn't support combo boxes yet.

Reply:
Fixed by just adding another rows. Will get the job done for now until we can do a documentation sprint.